.github/ISSUE_TEMPLATE/release.md

```diff
@@ -84,6 +84,8 @@ in the infrastructure repository. All of these tests can be run in parallel.
   - [ ] The "Time behind external source" dashboard panel in Grafana should have remained
     at 0ms or similar for the entirety of the run.
 
+- [ ] Verify the `chaos_run` container for each of the chaos tests exits successfully.
+
```

Comment:
how long should the chaos-run workflow run for? Is it worth this saying that it will finish in N hours and the instance can be destroyed then/adding a checkbox to delete the instance sooner then when the final release is getting cut?

How does one verify that? Do you need to ssh into the instance and check the logs? Could you enhance this comment by explaining the steps.